cdhweb/static_src/global/components/standard-hero.scss

```diff
@@ -0,0 +1,105 @@
+// has `.grid-standard`
+.standard-hero {
+  --title-top-space: 40px;
+
+  margin-block-end: 56px;
+
+  // t = title
+  // d = description
+  // i = image
+
+  // Layout if there is _no_ image:
+  grid-template-areas:
+    't t t t t t t t t t . .'
+    '. d d d d d d d d d . .';
+
+  @include sm {
+    margin-block-end: 120px;
+  }
+
+  @include md {
+    --title-top-space: 64px;
+
+    grid-template-areas:
+      't t t t t . i i i i i i'
+      '. d d d d d i i i i i i';
```

Comment:
is this right? feels like those `i` areas shouldn't be there if there's no image...

Reply:
Whoops, no. Good catch.